json/wnycstudios-gray.json

```diff
@@ -0,0 +1,27 @@
+{
+  "brand": "WNYC Studios",
```

Comment:
You should probably give this a different name since there’s already a WNYC Studios theme